blocks/header/header.css

```diff
@@ -1,275 +1,505 @@
+  text-decoration: none;
+  color: inherit;
+}
+
+.global-navigator .global-section-item a:hover {
+  color: var(--theme-dark-blue);
+}
+
+.global-navigator .global-section-item:not(:last-child) a {
+  border-right: var(--line-width-xxs) solid var(--normal-grey);
+}
+
+/* Copied accordian collapsable from hlx */
+
+.accordion {
+  font-size: var(--font-size-13px);
+  color: var(--normal-grey);
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.accordion a {
+  text-decoration: none; /* Remove underline */
+  color: inherit; /* Inherit color from parent */
+  user-select: none;
+}
+
+.accordion details summary {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
   list-style: none;
-  padding-left: 0;
-  font-size: var(--body-font-size-s);
-  font-weight: 500;
+  overflow: auto;
+  transition: background-color var(--time-xxs);
 }
 
-header nav .nav-sections ul > li {
-  font-weight: 700;
+.accordion details .accordion-item-label {
+  padding-top: var(--space-xxs2);
+  padding-bottom: var(--space-xxs2);
+  border-bottom: var(--line-width-xxs) solid var(--light-grey);
 }
 
-header nav .nav-sections ul > li > ul {
-  margin-top: 0;
+/* stylelint-disable-next-line no-descending-specificity */
+.accordion details a, .accordion-item-expand {
+  margin-left: var(--space-s);
+  margin-right: var(--space-s);
 }
 
-header nav .nav-sections ul > li > ul > li {
-  font-weight: 500;
+.accordion details .accordion-item-expand {
+  font-weight: var(--font-weight-600);
+  font-size: large;
+  color: var(--theme-dark-blue);
 }
 
-@media (width >= 900px) {
-  header nav .nav-sections {
-    display: block;
-    visibility: visible;
-    white-space: nowrap;
-  }
+.accordion details:not(:has(.accordion-subitem)) .accordion-item-expand {
+  display: none; /* for no sub items, hide the plus symbol */
+}
 
-  header nav[aria-expanded="true"] .nav-sections {
-    align-self: unset;
+.accordion details[open] .accordion-item-label {
+  border-bottom: none;
+  
+  a {
+    color: var(--theme-dark-blue);
   }
+}
+
+.accordion details:not(:has(.accordion-subitem)) .accordion-item-label {
+  border-bottom: var(--line-width-xxs) solid var(--light-grey);
+}
+
+.accordion details .accordion-subitem {
+  padding-top: var(--space-xxs2);
+  padding-bottom: var(--space-xxs2);
+  border-bottom: var(--line-width-xxs) solid var(--light-grey);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
 
-  header nav .nav-sections .nav-drop {
-    position: relative;
-    padding-right: 16px;
-    cursor: pointer;
+.accordion details .accordion-subitem .new-item-logo{
+  width: var(--space-ll);
+  height: var(--space-2);
+  margin-right: var(--space-xs);
+}
+
+/* Media queries starts here */
+
+@media screen and (width >= 48rem) {
+  /* CSS styles to apply when the screen width is more than 768px or 48rem */
+  .sidepanel {
```

Comment:
We should qualify the selectors with something like `.block header` etc. to ensure they are applicable only inside headers.

Reply:
Sure Satya, noted this will add this refactoring in the upcoming PR

Reply:
will be taken care in #33 PR